services/common/src/components/projectSummary/ProjectSubmissionStatusPage.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { Link, useParams } from "react-router-dom";
+import { Row, Col, Typography, Button } from "antd";
+import CheckCircleOutlined from "@ant-design/icons/CheckCircleOutlined";
+import ArrowLeftOutlined from "@ant-design/icons/ArrowLeftOutlined";
+import CloseCircleOutlined from "@ant-design/icons/CloseCircleOutlined";
+import LinkButton from "../common/LinkButton";
+import { ENVIRONMENT } from "@mds/common/constants";
+import * as API from "@mds/common/constants/API";
+import { AMS_STATUS_CODE_FAIL } from "@mds/common/constants";
+
+export const ProjectSubmissionStatusPage = () => {
```

Comment:
Could we add in a snapshot test for this component?

Reply:
This has been added now.